src/modules/redesign/parsers/profile.ts

```diff
@@ -146,7 +148,7 @@ export default <RedesignParser<ProfileWindow>>(({ LSSM, doc, href = '' }) => {
                                 prop => [
                                     prop.key.value,
                                     prop.value.type === 'UnaryExpression'
-                                        ? parseInt(
+                                        ? parseFloat(
```

Comment:
Are there really awards with floats? Or just prevention?

Reply:
This is the detection of building latitudes and longitudes. As those can be floats, this is a bug fix.